dlpar/dlpar_api/api.py

```diff
@@ -451,3 +629,552 @@ def set_virtual_proc_and_proc_units(self, linux_machine, ideal_procs,
                                     (curr_proc_units - proc_units_to_remove))
                 self.log.check_log(r_msg, r_condition)
                 curr_proc_units -= proc_units_to_remove
+
+
+class DedicatedCpu(TestCase):
+    """
+    DLPAR Dedicated CPU test case.
+
+    The test procedure:
+     1 - Get how many dedicated processors to test;
+     2 - Check and set the test environment for both linux partitions;
+     3 - Test (see run_test())
+
+    Everything is fine if we don't have troubles with the HMC and the linux
+    partitions are recognizing all added/removed cpus
+    (using dmesg and /var/log/messages).
+    """
+
+    def __init__(self, config_payload, log='dedicated_cpu.log'):
+
+        TestCase.__init__(self, log, 'Dedicated CPU', config_payload)
+
+        # Get test configuration
+        self.quant_to_test = 2
+        self.quant_to_test = config_payload.get('ded_quantity_to_test')
+        self.sleep_time = 60
+        self.sleep_time = config_payload.get('sleep_time')
```

Comment:
These variables are assigned values in succession. The first assignment is unnecessary.

Reply:
The above changes are addressed in the latest commit